charmcraft/extensions/__init__.py

```diff
@@ -39,3 +40,5 @@
     "register",
     "unregister",
 ]
+
+register("flask-framework", FlaskFramework)
```

Comment:
Please do two things here:

1. Export the `FlaskFramework` extension so it's available as `charmcraft.extensions.FlaskFramework`
2. Move this next to the other `register`.